setup_builder.py

```diff
@@ -57,6 +57,7 @@ class BuilderSection:
     """Encapsulates the `BUILDER_SECTION_NAME` section & checks for required/invalid fields."""
 
     pypi_name: str
+    package_paths: str = ""
     python_min: str  # python_requires
     python_max: str = ""  # python_requires
```

Comment:
Optional `dataclass` members have to be listed after required ones. There should be some kind of linting/runtime error.

Reply:
This would be caught by https://github.com/WIPACrepo/wipac-dev-py-setup-action/issues/4